be/src/column/const_column.h

```diff
@@ -78,7 +78,12 @@ class ConstColumn final : public ColumnFactory<Column, ConstColumn> {
 
     void reserve(size_t n) override {}
 
-    void resize(size_t n) override { _size = n; }
+    void resize(size_t n) override {
+        if (_size == 0) {
+            _data->resize(1);
```

Comment:
why we need call resize when _size == 0

Reply:
I need resize column to keep same rows when fill default value